src/components/Home/RecentlyPublished.tsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link'
+import { ReactElement, useEffect, useState } from 'react'
+import { FilterByTypeOptions } from '../../models/SortAndFilters'
+import { getRecentlyPublishedAssets } from '../../util/aquarius'
+import Assetlist from '../Assetlist'
+import Loader from '../atoms/Loader'
+import { filterAssetMetadata, SearchResults } from '../Search'
+import HomeSection from './HomeSection'
+import styles from './RecentlyPublished.module.css'
+
+export default function RecentlyPublished({
+  size = 10
+}: {
+  size?: number
+}): ReactElement {
+  const [recentAssets, setRecentAssets] = useState<SearchResults>()
+
+  useEffect(() => {
+    const loadPopularCategories = async () => {
+      const data = await getRecentlyPublishedAssets(size)
+      setRecentAssets(filterAssetMetadata(data))
+    }
+    loadPopularCategories()
+  }, [size])
+
+  return (
+    <HomeSection title="Latest Services">
+      {recentAssets?.metadata ? (
+        <div>
+          <Assetlist assets={recentAssets.metadata} />
+          <Link
+            href={{
+              pathname: '/search',
+              query: { type: FilterByTypeOptions.All, page: 1 }
+            }}
+          >
+            <a>
+              <p className={styles.browseAll}>Browse all services</p>
+            </a>
```

Comment:
issue/question(blocking): why is the additional `<a>` tag necessary?

Reply:
I don't think it's strictly necessary but the documentation always use it in the examples. I think it's for semantic HTML and maybe for SEO reasons. Should I remove it?